src/webauth/index.ts

```diff
@@ -180,33 +80,20 @@ class WebAuth {
    * @param {Bool} [parameters.federated] Optionally remove the IdP session.
    * @param {String} [parameters.customScheme] Custom scheme to build the callback URL with.
    * @param {Object} options Other configuration options.
-   * @param {String} [options.skipLegacyListener] Whether to register the event listener necessary for the SDK to work on iOS <11 or not. Defaults to `false`.
    * @returns {Promise}
    * @see https://auth0.com/docs/logout
    *
    * @memberof WebAuth
    */
-  clearSession(
-    parameters: ClearSessionParameters = {},
-    options: ClearSessionOptions = {}
-  ) {
-    const { client, agent, domain, clientId } = this;
-    if (Platform.OS == 'android') {
-      return agent.logout(
-        { clientId, domain },
-        {
-          customScheme: parameters.customScheme,
-          federated: parameters.federated,
-        }
-      );
-    } else {
-      const logoutParameters: any = parameters;
-      logoutParameters.clientId = clientId;
-      logoutParameters.returnTo = callbackUri(domain, parameters.customScheme);
-      logoutParameters.federated = parameters.federated || false;
-      const logoutUrl = client.logoutUrl(logoutParameters);
-      return agent.show(logoutUrl, false, options.skipLegacyListener, true);
-    }
+  clearSession(parameters: ClearSessionParameters = {}) {
+    const { agent, domain, clientId } = this;
+    return agent.logout(
+      { clientId, domain },
+      {
+        customScheme: parameters.customScheme,
```

Comment:
`customScheme` is part of the options for `authorize()` (see https://github.com/auth0/react-native-auth0/blob/vnext/src/webauth/index.ts#L77), but here is part of the parameters. Should it be a parameter or an option?

Reply:
`agent.logout` has different API than what we expose outside. `agent.login` and `agent.logout` take `clientId` and `domain` as parameters and all the other as options. The external API still remains the same.

Reply:
Looks like this has been always the case, fixed this as a part of the major release and added it to breaking change notes ([commit 1](https://github.com/auth0/react-native-auth0/pull/654/commits/d91096675b9d33efa7a9180d02447f028b1dfd14), [commit 2](https://github.com/auth0/react-native-auth0/pull/654/commits/2ab4104de3edf1646a055485ef9b8974ddfeead9))